nemo/src/execution/execution_engine.rs

```diff
@@ -120,11 +120,13 @@ impl<Strategy: RuleSelectionStrategy> ExecutionEngine<Strategy> {
                         .get(predicate)
                         .cloned()
                         .unwrap_or_else(|| vec![LogicalTypeEnum::Any; arity]);
-                    let reader = DSVReader::dsv(*file.clone(), *delimiter, logical_types);
+                    let reader = DSVReader::dsv(file.to_path_buf(), *delimiter, logical_types);
 
                     TableSource::FileReader(Box::new(reader))
                 }
-                DataSource::RdfFile(_) => todo!("RDF data sources are not yet implemented"),
+                DataSource::RdfFile(file) => {
```

Comment:
Now that we have a concrete implementation `RdfFile` seems a bit under-specified as an enum entry.

Reply:
Why? For N-Triples, there's no degrees of freedom anyways. We definitely want to overhaul the whole `@source` syntax (to, e.g., allow for different delimiters or escape characters in DSV files), so there's certainly an opportunity to change how we handle RDF input as well, but for now, this is what rulewerk does as well.